src/triage/component/architect/label_generators.py

```diff
@@ -60,7 +85,7 @@ def _create_labels_table(self, labels_table_name):
             label_name varchar(30),
             label_type varchar(30),
             label int
-        )'''.format(labels_table_name))
+        );'''.format(labels_table_name))
 
     def generate_all_labels(
```

Comment:
`generate_all_labels` and `_create_labels_table` should be in the base class, since their functionality is agnostic to the subclass' specifics.

We should probably rename this class to `InspectionsLabelGenerator` for clarity.